modules/gapi/misc/python/python_bridge.hpp

```diff
@@ -190,6 +192,134 @@ class GAPI_EXPORTS_W_SIMPLE GArrayT
     Storage m_arg;
 };
 
+namespace gapi {
+
+class GAPI_EXPORTS_W_SIMPLE GOutputs
```

Comment:
What it this and should it be in `cv::gapi`? I believe the namespace should indicate it is some internal thing.

Reply:
No problem, it can be in `wip` namespace